### ISSUE TITLE
Generalize whiptail usage to all platforms

### ISSUE THREAD

**Issue by tlaurion** [linuxboot/heads#477]
I think it might be a good time to replace generic-init by gui-init, and let the board configuration options define if FBWhiptail is desired but default to whiptail. I think that it would be a good base in improving the UX for everyone.

@kylerankin @osresearch @flammit : What do you think?

**Comment by tlaurion** [comment]
@merge : 
Yes. But each board should be validated individually to have proper Linux config FB support first from people having the models.

For the x230 board to become compatible with FBWhiptail, here were the [required changes](https://github.com/osresearch/heads/commit/98598e7f2f27eeb831a66eeb3f50d598c9ff50d0).
To have Whiptail (console based) instead of FBwhiptail (FB based):

```
#Whiptail
CONFIG_SLANG=y
CONFIG_NEWT=y
#FBWhiptail
#CONFIG_CAIRO=y
#CONFIG_FBWHIPTAIL=y

#BOOT SCRIPT SELECTION
#export CONFIG_BOOTSCRIPT=/bin/generic-init
export CONFIG_BOOTSCRIPT=/bin/gui-init
#export CONFIG_BOOTSCRIPT_NETWORK=/bin/network-init-recovery
```
This is the list of board owners i've gathered from board config changes "blames". Can you test the above changes on the board config on those boards and adapt linux config files to activate FB and report?

@osresearch : leopard, tioga
@vejmarie : r630, s2600wf
@jgrip @flammit @osresearch : x220

**Comment by osresearch** [comment]
leopard/tioga have no video cards at all, r630/s2600wf are almost always headless.

**Comment by tlaurion** [comment]
So something like for the kgpe-d16 for all those.

On February 8, 2019 4:17:05 PM EST, Trammell Hudson <notifications@github.com> wrote:
>leopard/tioga have no video cards at all, r630/s2600wf are almost
>always headless.
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/osresearch/heads/issues/477#issuecomment-461949793

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.
